src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
+
+const useStyles = makeStyles(
+  ({ palette: { secondary, warning }, spacing }) => ({
+    icon: {
+      height: spacing(2),
+      width: spacing(2),
+      marginRight: spacing(0.5),
+    },
+    iconWrapper: {
+      display: 'flex',
+      alignItems: 'center',
+      marginBottom: spacing(),
+    },
+    chip: {
+      borderRadius: '4px',
+      color: 'white',
+      backgroundColor: ({ value }: { value: string }) => {
```

Comment:
Ok so using property functions like this actually creates a new css class for each component that's mounted. Because of this I've come to the opinion that it's just easier to use inline `styles` instead of this. It's also easier on TypeScript, as you've seen a little bit of 😄

Reply:
That is no bueno (but good to know)! I'm going inline.